users/validator.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+REGEX_EMAIL    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+REGEX_PASSWORD = '^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,}$'
+
+def is_password(password):
```

Comment:
`is_password` 보다 `password_validation`이 더 나을것 같습니다. 단순히 비밀번호인지 판단하는게 아니라 비밀번호의 정규표현식을 검증하는 함수기 때문입니다.
```suggestion
def password_validation(password):
```